cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala

```diff
@@ -18,139 +18,136 @@ package com.rackspace.com.papi.components.checker.cli
 import javax.xml.transform._
 import javax.xml.transform.stream._
 
+import java.io.File
+import java.io.PrintStream
+import java.io.InputStream
+
 import com.rackspace.com.papi.components.checker.Config
 import com.rackspace.com.papi.components.checker.util.URLResolver
 import com.rackspace.com.papi.components.checker.wadl.WADLCheckerBuilder
+
 import org.clapper.argot.ArgotConverters._
 import org.clapper.argot.{ArgotParser, ArgotUsageException}
 
+import com.martiansoftware.nailgun.NGContext
+
 object Wadl2Checker {
   val title = getClass.getPackage.getImplementationTitle
-
   val version = getClass.getPackage.getImplementationVersion
 
-  val parser = new ArgotParser("wadl2checker", preUsage=Some(s"$title v$version"))
+  def parseArgs(args : Array[String], base : String,
+                in : InputStream, out : PrintStream, err : PrintStream) : Option[(Source, Result, Config)] = {
 
-  val removeDups = parser.flag[Boolean] (List("d", "remove-dups"),
-                                         "Remove duplicate nodes. Default: false")
+    val parser = new ArgotParser("wadl2checker", preUsage=Some(s"$title v$version"))
```

Comment:
Well, I rewrote the tests from scratch since the implementation changed dramatically 🤓